R/pr.R

```diff
@@ -491,11 +492,11 @@ pr_view <- function(number = NULL, target = c("source", "primary")) {
 #' @export
 #' @rdname pull-requests
 pr_pause <- function() {
-  # intentionally naive selection of target repo
-  tr <- target_repo(github_get = FALSE, ask = FALSE)
+  cfg <- github_remote_config(github_get = NA)
```

Comment:
This seems like a change, but isn't really because the call to `git_default_branch()` on line 498 would have used `github_get = NA`.